extension/resources/templates/report_failed_method_table.liquid

```diff
@@ -0,0 +1,42 @@
+<table border="1">

```

Comment:
Table is not the recommended way of displaying data. Use div instead when you have bootstrap ready.